src/components/search-form.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import UserInfo from '../stores/user-info';
+import UserAction from '../actions/user-action';
+
+export default class SearchForm extends React.Component {
+    constructor (props) {
+        super(props);
+
+        this.state = {
+            user: {
+                info:    {
+                    name: '',
```

Comment:
I asked you to add one more advanced functionality:
to highlight user's info parts that match searching criteria.

There is no infor about this functionality in the issue #11